pkg/registry-disk/registry-disk.go

```diff
@@ -20,257 +20,177 @@
 package registrydisk
 
 import (
-	"crypto/rand"
-	"encoding/base64"
+	"errors"
 	"fmt"
-	"strconv"
+	"os"
 
 	"github.com/jeevatkm/go-model"
 
 	kubev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/labels"
 
 	"kubevirt.io/kubevirt/pkg/api/v1"
-	"kubevirt.io/kubevirt/pkg/kubecli"
+	diskutils "kubevirt.io/kubevirt/pkg/ephemeral-disk-utils"
 	"kubevirt.io/kubevirt/pkg/precond"
 )
 
-const registryDiskV1Alpha = "ContainerRegistryDisk:v1alpha"
+const registryDiskV1Alpha = "RegistryDisk:v1alpha"
 const defaultIqn = "iqn.2017-01.io.kubevirt:wrapper/1"
 const defaultPort = 3261
 const defaultPortStr = "3261"
+const filePrefix = "disk-image"
 const defaultHost = "127.0.0.1"
 
-func generateRandomString(len int) (string, error) {
-	bytes := make([]byte, len)
-	_, err := rand.Read(bytes)
-	if err != nil {
-		return "", err
-	}
-	return base64.URLEncoding.EncodeToString(bytes), nil
+var registryDiskOwner = "qemu"
+
+var mountBaseDir = "/var/run/libvirt/kubevirt-disk-dir"
+
+func generateVMBaseDir(vm *v1.VirtualMachine) string {
+	domain := precond.MustNotBeEmpty(vm.GetObjectMeta().GetName())
+	namespace := precond.MustNotBeEmpty(vm.GetObjectMeta().GetNamespace())
+	return fmt.Sprintf("%s/%s/%s", mountBaseDir, namespace, domain)
+}
+func generateVolumeMountDir(vm *v1.VirtualMachine, diskCount int) string {
+	baseDir := generateVMBaseDir(vm)
+	return fmt.Sprintf("%s/disk%d", baseDir, diskCount)
 }
 
 func k8sSecretName(vm *v1.VirtualMachine) string {
```

Comment:
This seems to be unused now.

Reply:
ah, good catch. I stripped out the functionality that used that